### DISCUSSION TITLE
Possible bug: Intense drop of fps when armature is collinding with another object's collision bounds

### DISCUSSION THREAD

**Discussion by arch-sketch** [UPBGE/upbge#1475] [Ideas/Features for UPBGE 0.3+]
Im not sure if this is a bug or if its something Im doing wrong, but in testing a character with some animations in game, I found that having an object that is somehow in contact with the armature (with a collision type other than non collision), will make the physics meter go to 97% and fps drop to around 1. 

https://we.tl/t-lhZUJUHZva

Press P to play, and the frames should drop to almost zero instantly. If you delete the cube that is around the animated character , you will get normal fps, but it will drop again as soon as the player's own collision box comes in contact with the armature, if you move really close to it (wasd to move). The other character, in tpose, also has an armature, but we can go through it with no problems, unlike the one being animated. I should say that I have animated characters in other test files that work well with their collision boxes. The only thing I believe is different about this one is the fact that there are 2 other objects with armatures (sword with another armature with one bone, same for the scabbard) that both have a child of bone constraint targeted to the character's armature. Theres also a cone that is parented to the head bone in the character armature (that was going to be a sensor object)

**Comment by wetoo-cando** [comment]
@arch-sketch did the collision detection actually work for you?
